.github/workflows/integration-tests.yml

```diff
@@ -244,7 +242,7 @@ jobs:
             discovery-service:
                 image: ghcr.io/balhar-jakub/discovery-service:${{ github.run_id }}-${{ github.run_number }}
             gateway-service:
-                image: ghcr.io/balhar-jakub/gateway-service:${{ github.run_id }}-${{ github.run_number }}
+                image: ghcr.io/balhar-jakub/gateway-service:7749575670-947
```

Comment:
this will be merged?

Reply:
now sure how we can get the lastest build from v2, do you have some idea?

Reply:
I assume this is just an image tag, I'm just thinking out loud, but could we update v2 branch to publish a `latest` tag?

Reply:
replaced with latest-v2